core/modules/mysql/MySqlConnection.cc

```diff
@@ -200,6 +200,7 @@ MYSQL* MySqlConnection::_connectHelper() {
         janitor.reset(new MySqlThreadJanitor);
     }
     unsigned long clientFlag = CLIENT_MULTI_STATEMENTS;
+	mysql_options( m, MYSQL_OPT_LOCAL_INFILE, 0 );
```

Comment:
Your editor inserts TABs again, replace with spaces. Spaces are not needed after/before parenthesis.